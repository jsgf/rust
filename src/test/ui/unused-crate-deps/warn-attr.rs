// Check for unused crate dep, no path

// edition:2018
// check-pass
// aux-crate:bar=bar.rs
// no-prefer-dynamic
// ignore-tidy-linelength

#![warn(unused_crate_deps)]
//~^ WARNING External crate `bar` unused in `warn_attr`. Remove the dependency or add `use bar as _;`.

fn main() {}
