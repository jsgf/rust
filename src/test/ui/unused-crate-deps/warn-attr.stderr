warning: External crate `bar` unused in `warn_attr`. Remove the dependency or add `use bar as _;`.
  --> $DIR/warn-attr.rs:9:1
   |
LL | #![warn(unused_crate_deps)]
   | ^
   |
note: the lint level is defined here
  --> $DIR/warn-attr.rs:9:9
   |
LL | #![warn(unused_crate_deps)]
   |         ^^^^^^^^^^^^^^^^^

